lib/slack-handler.js

```diff
@@ -26,17 +34,21 @@ var SLACK_EVENTS = {
 
     // Prepare IRC connection
     var user = context.slack.client.data.getUserById(context.slack.client.rtm.activeUserId);
+    context.logger.debug('IRC config -> '+user.profile.title);
     var userConfig = HELPERS.parseConfig(user.profile.title);
     // TODO: Fetch channel config from channel purpose
     var userChannels = _.chain(context.slack.client.data.channels)
                     .filter('is_member')
                     .map(HELPERS.getIrcChannelName.bind(null, context))
+                    .filter(function(channel) {
+                      return channel.name !== '';
+                    })
                     .valueOf();
 
     context.irc.options = {
       server: userConfig.server,
       port: userConfig.port || 6667,
-      nick: user.name,
+      nick: userConfig.userName || user.name,
```

Comment:
Or is this confusion about what the `userName` in the examples is? It's just used to demonstrate the passing of options to `node-irc`, which takes `userName` as an option (http://node-irc.readthedocs.io/en/latest/API.html#irc.Client). In IRC username and nickname are different things, and should be set separately.